exercises/practice/list-ops/ListOpsTest.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+/*
+ * By adding type hints and enabling strict type checking, code can become
+ * easier to read, self-documenting and reduce the number of potential bugs.
+ * By default, type declarations are non-strict, which means they will attempt
+ * to change the original type to match the type specified by the
+ * type-declaration.
+ *
+ * In other words, if you pass a string to a function requiring a float,
+ * it will attempt to convert the string value to a float.
+ *
+ * To enable strict mode, a single declare directive must be placed at the top
+ * of the file.
+ * This means that the strictness of typing is configured on a per-file basis.
+ * This directive not only affects the type declarations of parameters, but also
+ * a function's return type.
+ *
+ * For more info review the Concept on strict type checking in the PHP track
+ * <link>.
+ *
+ * To disable strict typing, comment out the directive below.
+ */
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\ExpectationFailedException;
+
+class ListOpsTest extends PHPUnit\Framework\TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        require_once 'ListOps.php';
+    }
+
+    /**
+     * @testdox append entries to a list and return the new list -> empty lists
```

Comment:
I love this! It looks even better with a capital letter at the beginning of the sentence. And I suggest adding the UUID of the test case, [like here](https://github.com/tomasnorre/exercism-php/blob/c16ce6546760368bcb588ae265cfae6d2f2be38f/exercises/practice/nucleotide-count/NucleotideCountTest.php#L48):

```suggestion
     * @testdox Append entries to a list and return the new list -> empty lists
     * uuid: 485b9452-bf94-40f7-a3db-c3cf4850066a
```

Reply:
I just copy-pasted those from the JavaScipt track.

The `@testdox` matches the description in the `tests.toml` file, do you think I should still diverge from this description and use a capital letter ?

Reply:
> The description in the problem spec is written to be used like `sprintf('It %s', $description)`.

So maybe we should update this is the test runner instead ? So we can keep this consistent with the toml.

Reference: https://github.com/exercism/php-test-runner/blob/61252274d05daf021fa70c6166edc1b9a0d1ce63/junit-handler/src/Handler.php#L160

Or you prefer we do this on a test by test basis ?

Reply:
I prefer having that in the test. Because this makes PHPUnit output the testdox nicely on the command line when working with the CLI.

I put some hope onto [tomasnorre's test generator](https://github.com/tomasnorre/exercism-tests-generation) to take care of that. But for now, I would be happy if you do it. I will merge with or withour capitals...